draft-ietf-rtcweb-jsep.xml

```diff
@@ -1878,14 +1878,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <xref target="RFC4566"></xref>, Section 5.1</t>
 
             <t>The second SDP line MUST be an "o=" line, as specified
-            in
-            <xref target="RFC4566"></xref>, Section 5.2. The value of
-            the &lt;username&gt; field SHOULD be "-".
-            <xref target="RFC3264" /> requires that the &lt;sess-id&gt;
-            be representable as a 64-bit signed integer. It is
-            RECOMMENDED that the &lt;sess-id&gt; be generated as a
-            64-bit quantity with the high bit being sent to zero and
-            the remaining 63 bits being cryptographically random. The
+            in <xref target="RFC4566"></xref>, Section 5.2. The value
+            of the &lt;username&gt; field SHOULD be "-".
+            <xref target="RFC3264" /> requires that the inital value
+            of the &lt;sess-id&gt; be representable as a 64-bit signed
+            integer less than (2**62)-1.  <xref target="RFC3264" />
+            requires that the two high-order bits of the
+            &lt;sess-id&gt; initially be set to zero. It is
+            RECOMMENDED that the remaining 62 bits are initially set
+            to a cryptographically random value. The
```

Comment:
I think that this is semantically correct, but in the style of JSEP, I would say.

"The sess-id MUST be representable by a 64-bit signed integer, and the initial value MUST be less than (2**62)-1, as required by RFC 3264. It is RECOMMENDED that this be constructed by generating a 64-bit quantity with the two highest bits being set to zero and the remaining 62 bits being cryptographically random."